src/main/java/ladder/domain/Line.java

```diff
@@ -0,0 +1,102 @@
+package ladder.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Line {
+    private List<Boolean> points = new ArrayList<>();
+    private int pointCount = 0;
+    private int widthLadderCount = 0;
+    private int countOfPerson = 0;
+    private static final String EMPTY_WIDTH_LADDER = "     ";
+    private StringBuilder sb = new StringBuilder();
+    private final RandomNumber randomNumber = new RandomNumber();
+
+
+    public Line(int countOfPerson) {
+        this.countOfPerson = countOfPerson;
+        pointCount = calculatePointByPeople();
+        addLine();
+    }
+
+    public void addWidthLine(int position) {
+        if (!checkAdjustLine(countOfPerson, position)) {
+            points.set(position, true);
+        }
+    }
+
+    private boolean checkAdjustLine(int countOfPerson, int position) {
+        return countOfPerson > 2 && position > 1 && points.get(position - 2);
+    }
+
+    private int calculatePointByPeople() {
+        widthLadderCount = countOfPerson - 1;
+        return widthLadderCount + countOfPerson;
+    }
+
+    private void drawOneLineLadder(int index) {
+        if (isHeightLadder(index)) {
+            sb.append("|");
+        }
+
+        if (!isHeightLadder(index) && isDrawOneWidthLadder(index)) {
+            sb.append("-----");
+        }
+
+        if (!isHeightLadder(index) && !isDrawOneWidthLadder(index)) {
+            sb.append(EMPTY_WIDTH_LADDER);
+        }
+    }
+
+    private boolean isDrawOneWidthLadder(int index) {
+        return points.get(index);
+    }
+
+    private boolean isHeightLadder(int index) {
+        return index % 2 == 0 ? true : false;
```

Comment:
return해주시는 값 자체가 boolean값이네용! 
아래처럼 표현이 가능 한 것 같습니다 :) 
```suggestion
       return index % 2 == 0;
```